packages/twenty-front/src/modules/object-record/cache/utils/getRecordNodeFromRecord.ts

```diff
@@ -79,7 +77,7 @@ export const getRecordNodeFromRecord = <T extends ObjectRecord>({
             getRecordConnectionFromRecords({
               objectMetadataItems,
               objectMetadataItem: oneToManyObjectMetadataItem,
-              records: value as ObjectRecord[],
+              records: (value || []) as ObjectRecord[],
```

Comment:
i understand the changes too, I'm not agaist them but we should add tests on these